packages/SwingSet/src/controller/controller.js

```diff
@@ -248,7 +251,7 @@ export async function makeSwingsetController(
       return result;
     } catch (cause) {
       if (!finished) {
-        const msg = `unfinished slog event ${startLabel} ${q(startProps || {})}`;
+        const msg = `unfinished slog event ${q(startLabel)} ${q(startProps || {})}`;
```

Comment:
This too seems to need an endLabel:

```suggestion
        const msg = `unfinished slog event ${q(startLabel || endLabel)} ${q(startProps || {})}`;
```